python/pyarrow/array.pxi

```diff
@@ -2638,20 +2651,66 @@ cdef class ListViewArray(Array):
         Examples
         --------
         >>> import pyarrow as pa
-        >>> values = [[1, 2], [3, 4, None, 6]]
-        >>> offsets = [0, None, 2]
+        >>> values = [1, 2, None, 3, 4]
+        >>> offsets = [0, 0, 1]
         >>> sizes = [2, 0, 4]
         >>> array = pa.ListViewArray.from_arrays(offsets, sizes, values)
-        >>> array.offsets
+        >>> array.sizes
         <pyarrow.lib.Int32Array object at ...>
         [
+          2,
           0,
-          0,
-          2
+          4
         ]
         """
         return pyarrow_wrap_array((<CListViewArray*> self.ap).sizes())
 
+    def flatten(self, pool=None):
```

Comment:
```suggestion
    def flatten(self, memory_pool=None):
```

We are not very consistent about it, but at least in this file we use `memory_pool` more than `pool` .. (something to clean up at some point)